ansible/roles/schulcloud-server-core/templates/svc.yml.j2

```diff
@@ -12,6 +12,10 @@ spec:
       targetPort: 3030
       protocol: TCP
       name: api
+    - port: 4030
+      targetPort: 4030
+      protocol: TCP
+      name: api-admin
```

Comment:
The api deployment has no api-admin port so cann be removed
```suggestion
```

Reply:
done